### ISSUE TITLE
`Video` field preview tries to preview invalid sources

### ISSUE THREAD

**Issue by Cube-Justin** [statamic/cms#9943]
### Bug description

I am currently loading a Vimeo video by using the `Video` field in the CMS.
When using a valid URL, such as: `https://vimeo.com/935832884` , the preview loads fine.
When inputting an invalid URL, such as: `blabla.test`, the preview is not loaded as would be expected.
However when inputting an Vimeo embed as a source, such as: 
```
<iframe src="https://player.vimeo.com/video/882020077?h=53a2cf6397" width="640" height="360" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe>
<p><a href="https://vimeo.com/882020077">FESTIVAL OF SLAPS</a> from <a href="https://vimeo.com/abdoucisse">Abdou Ciss&eacute;</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
```
The CMS still tries to load a preview, but this fails of course. The preview in this case is is an embed of the CMS:
<img width="873" alt="image" src="https://github.com/statamic/cms/assets/132989832/ab43f680-2b87-4c23-b04b-f690c72dcdb5">

### How to reproduce

1. Add a `Video` field to a blueprint
2. Insert the following text into the field:
```
<iframe src="https://player.vimeo.com/video/882020077?h=53a2cf6397" width="640" height="360" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe>
<p><a href="https://vimeo.com/882020077">FESTIVAL OF SLAPS</a> from <a href="https://vimeo.com/abdoucisse">Abdou Ciss&eacute;</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
```

### Environment

```yaml
Environment
Application Name: #confidential#
Laravel Version: 10.44.0
PHP Version: 8.3.4
Composer Version: -
Environment: production
Debug Mode: OFF
URL: #confidential#
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: DAILY
Mail: smtp
Queue: sync
Session: file

Sentry
Enabled: YES
Environment: production
Laravel SDK Version: 4.2.0
PHP SDK Version: 4.6.0
Release: NOT SET
Sample Rate Errors: 100%
Sample Rate Performance Monitoring: NOT SET
Sample Rate Profiling: NOT SET
Send Default PII: DISABLED

Statamic
Addons: 2
Antlers: runtime
Sites: 2 (Nederlands, Engels)
Stache Watcher: Disabled
Static Caching: Disabled
Version: 4.48.0 PRO

Statamic Addons
pecotamic/sitemap: 1.4.6
statamic/eloquent-driver: 3.1.1

Statamic Eloquent Driver
Asset Containers: file
Assets: eloquent
Blueprints: file
Collection Trees: eloquent
Collections: file
Entries: eloquent
Forms: eloquent
Global Sets: file
Global Variables: eloquent
Navigation Trees: eloquent
Navigations: file
Revisions: eloquent
Taxonomies: file
Terms: eloquent
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

Runtime (default)

**Comment by duncanmcclean** [comment]
You're not meant to copy HTML code into the Video Fieldtype - instead, you should paste in the URL of the video. For example, you'd just copy in this portion: `https://player.vimeo.com/video/882020077`

As [mentioned in the docs](https://statamic.dev/fieldtypes/video#usage), you can enter the URL of a video file, YouTube video or Vimeo video.

**Comment by Cube-Justin** [comment]
@duncanmcclean I understand that this is how the field is supposed to be used.
However, I think that this still should be marked as e `Bug` since the preview that is loaded when an invalid 'URL' is given.

This can result in some weird/hacky request to the server, which in our case was caught by the firewall on the server, and blocked the IP address of our client.
The request in this scenario:
`/admin/collections/pages/entries/%3Cdiv%20style=%22padding:56.17%%200%200%200;position:relative;%22%3E%3Ciframe%20src=%22https://player.vimeo.com/video/935832884?badge=0&amp;autopause=0&amp;player_id=0&amp;height:100%;%22%20title=%22Headervide`

So while on first sight this might seem like a small UX issue, I believe the consequences of this request can be critical in certain scenarios. Especially when the developers are not the ones using the CMS.

**Comment by duncanmcclean** [comment]
I've opened a pull request to fix that issue: #9944.
